data/kernels_and_extensions.yml

```diff
@@ -0,0 +1,30 @@
+Links:
+  Python: https://github.com/python/python
+  Matlab: https://github.com/mathworks/jupyter-matlab-proxy
+  R: https://github.com/IRkernel/IRkernel
+  Octave: https://github.com/Calysto/octave_kernel
+  Sage: https://github.com/sagemath/sage
+  gnuplot: https://github.com/has2k1/gnuplot_kernel
+  Mathematica: https://github.com/WolframResearch/WolframLanguageForJupyter
+  Cling: https://github.com/jupyter-xeus/xeus-cling
+  evcxr: https://github.com/evcxr/evcxr
+  Gophernotes: https://github.com/gopherdata/gophernotes
+  Calysto Scheme: https://github.com/Calysto/calysto_scheme
+  Bash: https://github.com/takluyver/bash_kernel
+  JavaScript: https://github.com/n-riesco/ijavascript
+  Ruby: https://github.com/SciRuby/iruby
+  Ansible: https://github.com/ansible/ansible-jupyter-kernel
+  Cling: https://github.com/jupyter-xeus/xeus-cling
```

Comment:
Cling is already there, line 9 - please remove redundant entries. This list should be compact. Maybe sort alphabetically?

Reply:
Done.

Reply:
Alphabetically sorting isn't true anymore - please check.